src/foam/comics/DAOController.js

```diff
@@ -186,7 +186,7 @@ foam.CLASS({
         var sectionName = undefined;
         var hasManySections = false;
         var model = this.data.of.model_;
-        while ( model && model !== foam.core.FObject ) {
+        while ( model && model === foam.core.FObject ) {
```

Comment:
I think this defeats the purpose of this model parent iteration and looking for any associated sections. I think this might be something we're looking for:
```suggestion
        while ( model && model.id != 'foam.core.FObject' ) {
```

Reply:
Thought I submitted this comment yesterday :/